mlrun/k8s_utils.py

```diff
@@ -329,13 +331,38 @@ def get_project_vault_secret_name(
 
         return service_account.secrets[0].name
 
-    def get_project_secret_name(self, project):
+    def get_project_secret_name(self, project) -> str:
         return mlconfig.secret_stores.kubernetes.project_secret_name.format(
             project=project
         )
 
+    def get_auth_secret_name(self, username: str, access_key: str) -> str:
```

Comment:
See comment in the Secrets CRUD object - I think this does not belong here.

Reply:
same